UI/src/view/linear_asset/pavedRoadLayer.js

```diff
@@ -0,0 +1,18 @@
+(function (root) {
```

Comment:
This file is not needed , the unique method override here has the same code on LinearAssetLayer 
```
  this.getLayerStyle = function(feature)  {
      return style.browsingStyleProvider.getStyle(feature, {zoomLevel: me.uiState.zoomLevel});
  };
```